src/assertError.ts

```diff
@@ -0,0 +1,14 @@
+import { Steps } from './assertOrder'
+
+export default class AssertError extends Error {
```

Comment:
ES6 default export as a number of drawbacks. It is better to just use named exports.